uaclient/tests/test_cli.py

```diff
@@ -279,7 +280,13 @@ def test_url_error_handled_gracefully(
         assert "" == out
         assert "{}\n".format(status.MESSAGE_CONNECTIVITY_ERROR) == err
         error_log = caplog_text()
-        assert " [Errno -2] Name or service not known" in error_log
+        expected_log = "Check your internet connection and try again. "
```

Comment:
A nit, but we could just have the expected log message (or message suffix) as parameters to the test.

Reply:
works for me.